.vimrc

```diff
@@ -84,8 +84,10 @@ set cursorline
 
 ""------------ Mappings --------------"
 "" Create tree navigation
-map <C-n> :vsp . <bar> vertical resize 30 <cr>
+map <C-n> :Lexplore <CR>
```

Comment:
Almost. It uses Explorer, but it makes netrw toggleable by default, we just need to set a default size, I set 28, but it could be 30.